api/v1beta1/flowcollector_types.go

```diff
@@ -485,26 +492,26 @@ type FlowCollectorLoki struct {
 	QuerierURL string `json:"querierUrl,omitempty"`
 
 	//+kubebuilder:validation:optional
-	// statusURL specifies the address of the Loki /ready /metrics /config endpoints, in case it is different from the
-	// Loki querier URL. If empty, the QuerierURL value will be used.
+	// statusURL specifies the address of the Loki `/ready`, `/metrics` and `/config` endpoints, in case it is different from the
+	// Loki querier URL. If empty, the `querierURL` value will be used.
 	// This is useful to show error messages and some context in the frontend.
 	// When using the Loki Operator, set it to the Loki HTTP query frontend service, for example
 	// https://loki-query-frontend-http.netobserv.svc:3100/.
-	// statusTLS configuration will be used when statusUrl is set.
+	// `statusTLS` configuration will be used when `statusUrl` is set.
 	StatusURL string `json:"statusUrl,omitempty"`
 
 	//+kubebuilder:default:="netobserv"
-	// tenantID is the Loki X-Scope-OrgID that identifies the tenant for each request.
+	// tenantID is the Loki `X-Scope-OrgID` that identifies the tenant for each request.
 	// When using the Loki Operator, set it to `network`, which corresponds to a special tenant mode.
 	TenantID string `json:"tenantID,omitempty"`
 
 	// +kubebuilder:validation:Enum:="DISABLED";"HOST";"FORWARD"
 	//+kubebuilder:default:="DISABLED"
-	// AuthToken describe the way to get a token to authenticate to Loki.
-	// DISABLED will not send any token with the request.
-	// HOST - <i>deprecated (*)</i> - will use the local pod service account to authenticate to Loki.
-	// FORWARD will forward the user token for authorization.
-	// When using the Loki Operator, this should be set to `FORWARD`.
+	// AuthToken describe the way to get a token to authenticate to Loki.<br>
+	// - `DISABLED` will not send any token with the request.<br>
+	// - `FORWARD` will forward the user token for authorization.<br>
+	// - `HOST` - <i>deprecated (*)</i> - will use the local pod service account to authenticate to Loki.<br>
```

Comment:
same here: deprecated or unsupported for 1.3+ ?

Reply:
Here I think it's clearly deprecated: it used to be a supported feature, and is not anymore